internal/applicationsnapshot/report.go

```diff
@@ -235,6 +239,51 @@ func condensedMsg(results []evaluator.Result) map[string][]string {
 	return shortNames
 }
 
+func generateMarkdownSummary(r *Report) ([]byte, error) {
+	var markdownBuffer bytes.Buffer
+	markdownBuffer.WriteString("| Field     | Value |Status|\n")
+	markdownBuffer.WriteString("|-----------|-------|-------|\n")
+
+	var totalViolations, totalWarnings, totalSuccesses int
+	pr := r.toSummary()
+	for _, component := range pr.Components {
+		totalViolations += component.TotalViolations
+		totalWarnings += component.TotalWarnings
+		totalSuccesses += component.TotalSuccesses
+	}
+	type Field struct {
+		Name  string
+		Value string
+	}
+	fields := []Field{
+		{"Time", r.created.UTC().Format("2006-01-02 15:04:05")},
+		{"Name", " "},
+		{"Successes", fmt.Sprint(totalSuccesses)},
+		{"Failures", fmt.Sprint(totalViolations)},
+		{"Warnings", fmt.Sprint(totalWarnings)},
+		{"Result", fmt.Sprint(pr.Success)},
+	}
+
+	for _, field := range fields {
+		writeMarkdownField(&markdownBuffer, field.Name, field.Value)
+	}
+	return markdownBuffer.Bytes(), nil
+}
+
+func writeMarkdownField(buffer *bytes.Buffer, name, value string) {
+	var icon string
+	intVal, _ := strconv.Atoi(value)
+	icon = ":x:"
+	if (name == "Name" || name == "Time") ||
+		(name == "Successes" && intVal >= 1) ||
+		(name == "Failures" && intVal == 0) ||
+		(name == "Warnings" && intVal == 0) ||
+		(name == "Result" && value == "true") {
+		icon = ":white_check_mark:"
+	}
```

Comment:
Bit odd to convert values to string and then back to number to compare values. Could do something like this in `generateMarkdownSummary`:

```go

icon := func(val, treshold int) string {
	if val >= treshold {
		return ":white_check_mark:"
	}

	return ":x:"
}

writeMarkdownField(&markdownBuffer, "Successes", totalSuccesses, icon(totalSuccesses, 1))
```

And change `writeMarkdownField` to something like:

```go
func writeMarkdownField(buffer *bytes.Buffer, name string, value fmt.Stringer, icon string) {
```